.github/workflows/lint-and-test.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Run tests and report code coverage
         run: |
           # enable nightly features so that we can also include Doctests
-          RUSTC_BOOTSTRAP=1 cargo tarpaulin -o xml -o lcov -o html --doc --tests
+          RUSTC_BOOTSTRAP=1 cargo tarpaulin -o xml -o lcov -o html --doc --tests --all-features
```

Comment:
now we no longer test if it actually compiles/works _without_ the `cloudevents` feature, do we?

Reply:
This is test code coverage collection - makes most sense to run with all features.
I'd say we need something dedicated in addition now, to cover the matrix of possible features. Am looking into that atm...

Reply:
I'll take it you'll ping us when you have a solution for that in mind for another round of code review. :+1: